### DISCUSSION TITLE
CHANGELOG.md: missing infos about, e.g., 1.2.1

### DISCUSSION THREAD

**Discussion by goto40** [eclipse-langium/langium#1129] [Q&A]
I am missing the CHANGELOG for current versions? Is it maintained elsewhere? I checked https://github.com/langium/langium/blob/main/CHANGELOG.md

**Comment by spoenemann** [comment]
@msujew could you add that as well when you work on the v2.0.0 changelogs?

**Comment by msujew** [comment]
@goto40 Langium 1.2.1 only contained a minor bugfix that we needed in one of our projects, see https://github.com/langium/langium/pull/1043.
